cmd/neofs-adm/internal/modules/morph/root.go

```diff
@@ -111,6 +111,16 @@ var (
 		RunE: forceNewEpochCmd,
 	}
 
+	removeNodes = &cobra.Command{
+		Use:   "remove-nodes key1 [key2 [...]]",
+		Short: "Remove storage nodes from the netmap",
```

Comment:
Can we add a more detailed description for the command, clarifying it will remove the node from the candidate list and that it needs an Epoch tick to become the new netmap?

Reply:
Fixed